tokio/CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# 1.30.0 (August 6, 2023)
+
+### Fixed
+- sync: fix import style for `std::error::Error` (#5818)
+- util: fix broken intra-doc link (#5849)
+- process: fix `raw_arg` not showing up in docs (#5865)
+- ci: fix clippy warnings (#5891)
+- runtime: fix flaky test `wake_while_rt_is_dropping` (#5905)
+- rt(alt): fix a number of concurrency bugs (#5907)
+
+### Changed
+
+- ci: reenable semver check (#5845)
+- Speedup CI (#5691)
+- tokio: reduce LLVM code generation (#5859)
+- runtime: use `Arc::increment_strong_count` instead of `mem::forget` (#5872)
+- rt: use optional non-zero value for task `owner_id` (#5876)
+- poll: Do not clear readiness on short read/writes. (#5881)
+- sync: make `const_new` methods always available (#5885)
+- sync: avoid false sharing in mpsc channel (#5829)  
+### Added
+
+- sync: add `broadcast::Sender::new` (#5824)
+- net: implement `UCred` for espidf (#5868)
+- rt(alt): track which workers are idle. (#5886)
+- fs: add `File::options()` (#5869)
+- rt: add runtime ID (#5864)
+- test: fetch actions from mock handle before write (#5814)
+- rt: initial implementation of new threaded runtime (#5823)
+- time: implement extra reset variants for `Interval` (#5878)
+- rt: pop at least one task from inject queue (#5908)
+
+### Removed
+
+- tokio: removed unused `tokio_*` cfgs (#5890)
```

Comment:
It's worth mention that the entire `build.rs` is removed so compiling tokio should be faster.

Reply:
Does that go into that LOC or as a separate line? I assumed the commit names were the updates, or should I expand upon the commit name for #5890 ?

Reply:
I think it should go as a separate line.

Reply:
I think it should go as a separate line.